setup.py

```diff
@@ -30,7 +30,8 @@
             "solc-select>=v1.0.0b1",
             "openai",
             "pdoc",
-        ]
+        ],
+        "read-storage": ["solc-select>=v1.0.0b1", "web3>=6.0.0b"],
```

Comment:
Is solc-select a dependency? (I was thinking this might be useful to install a "fully featured" Slither with brew, but solc-select so far is not a sliither dependency)

Reply:
To use slither with Etherscan, I think solc-select is required